doc/tutorials/parallel.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
It should. But readthedocs hangs to execute MPI part of the tutorial even though mpi4py is installed. It's because of the intervention of two things: conda mpi4py package is somewhat broken (compared to linux libopenmpi package) and all the scripts that have mpi should be executed with mpiexec command.

Reply:
and the pip requirement for mpi4py also doesn't work on readthedocs?

Reply:
precisely